pkg/docker/docker.go

```diff
@@ -171,7 +188,7 @@ func (rco RunContainerOptions) asDockerCreateContainerOptions() docker.CreateCon
 	config := rco.asDockerConfig()
 	hostConfig := rco.asDockerHostConfig()
 	return docker.CreateContainerOptions{
-		Name:       "",
+		Name:       containerName("", ""),
```

Comment:
BTW, why we're generating names? Could we specify names explicitly? For example, "Builder container", "Runtime image container" etc.